cmake/Modules/GnuradioConfig.cmake.in

```diff
@@ -17,13 +17,7 @@
 # the Free Software Foundation, Inc., 51 Franklin Street,
 # Boston, MA 02110-1301, USA.
 
-INCLUDE(FindPkgConfig)
-INCLUDE(FindPackageHandleStandardArgs)
-
-# if GR_REQUIRED_COMPONENTS is not defined, it will be set to the following list (all of them)
-if(NOT GR_REQUIRED_COMPONENTS)
-  set(GR_REQUIRED_COMPONENTS RUNTIME ANALOG BLOCKS DIGITAL FFT FILTER PMT)
-endif()
```

Comment:
Removing this was "just" a test, we shouldn't actually remove all of it for compatibility for at least 3.8.

Reply:
We need to discuss/think about how to handle "old" and "modern" CMake. Or if we just require for 3.8 that downstream libraries/packages have to use modern CMake.

Reply:
From what I've learned/read it's near impossible to support the old variable exporting approach with targets. Basically one would have to render all include paths embedded in exported targets during generation time of the `GnuradioConfig.cmake`. I can't seem to find a way of doing that.

Reply:
I would like to stick with the 3.5.1 CMake version we currently have on master.

Reply:
...unless this totally screws up this PR.